recipes/graylog-enterprise/patches/graylog-server.conf.patch2

```diff
@@ -0,0 +1,9 @@
+--- graylog.conf.example        2021-03-31 10:00:49.785429250 -0500
++++ graylog.conf        2021-03-31 10:00:40.693473340 -0500
+@@ -686,3 +686,6 @@
+ # Setting this to an empty value, leaves it up to system libraries and the used JDK to chose a default.
+ # Default: TLSv1.2,TLSv1.3  (might be automatically adjusted to protocols supported by the JDK)
+ #enabled_tls_protocols= TLSv1.2,TLSv1.3
++
++# An absolute or relative path where scripts are permitted to be executed from.
```

Comment:
Let's avoid suggesting a relative path. When using the OS packages, the working dir of the server is `/`.

```suggestion
+# An absolute path where scripts are permitted to be executed from.
```